.github/workflows/e2e.istio.yaml

```diff
@@ -134,7 +136,13 @@ jobs:
   metrics-service:
     runs-on: ubuntu-16.04
     timeout-minutes: 60
-    name: MetricsService
+    name: Istio(${{ matrix.istio_version }})+MetricsService
```

Comment:
This includes meta exchange,  right?

Reply:
This is "Metrics Service", Metrics Service has nothing to do with Meta Exchange. Check line 171 "# disable the metadata-exchange extension intentionally to make sure metrics service doesn't rely on it".

As for Meta Exchange verification, it's in line 40-42